Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -301,29 +301,24 @@ of the `C3T3`. They can be removed using the function `remove_isolated_vertices(
 
 \section Mesh_3_section_interface Interface
 
+Since \cgal 5.6, this packages uses \ref bgl_namedparameters to set parameters. More details are provided in  \ref FromBoostNPtoCGALNP.
```

Comment:
``` suggestion
Since \cgal 5.6, this package uses \ref bgl_namedparameters to set parameters. More details are provided in \ref FromBoostNPtoCGALNP.
```